packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -304,7 +304,7 @@ export class PendingStateManager implements IDisposable {
         if (pendingState.clientSequenceNumber !== message.clientSequenceNumber) {
             // Close the container because this indicates data corruption.
             const error = new DataCorruptionError(
-                "Unexpected ack received",
```

Comment:
Can you change this from DataCorruptionError to DataProcessingError. This error doesn't definitely mean the data is corrput, only that it could be, and we generally use DataProcessingError for that

Reply:
Done